.ci.yaml

```diff
@@ -4414,6 +4414,7 @@ targets:
       tags: >
         ["devicelab", "hostonly", "windows"]
       task_name: plugin_test
+      test_timeout_secs: "3600"
```

Comment:
Do you know what is this test doing? would it be possible to shard it rather increasing the timeout?

Reply:
@stuartmorgan thoughts on this?

Reply:
It's running [three variants of plugin tests](https://github.com/flutter/flutter/blob/master/dev/devicelab/bin/tasks/plugin_test_windows.dart) (similar to [android](https://github.com/flutter/flutter/blob/master/dev/devicelab/bin/tasks/plugin_test.dart), [ios](https://github.com/flutter/flutter/blob/master/dev/devicelab/bin/tasks/plugin_test_ios.dart), [linux](https://github.com/flutter/flutter/blob/master/dev/devicelab/bin/tasks/plugin_test_linux.dart), and [macos](https://github.com/flutter/flutter/blob/master/dev/devicelab/bin/tasks/plugin_test_macos.dart)).

I'm not familiar with the details of the test infrastructure here; is `test_timeout_secs` a timeout that's being applied to the entire array of `TaskFunction`s here instead of handling each one separately?

Each of those calls `PluginTest` calls should stand alone, so any way of restructuring the list of calls that improves timeout handling, sharding, etc. should be fine.

Reply:
Based on the linked variants, I recommend we shard this target to three if the total execution is close to 1h. Say
`Windows pluign_test`
`Windows pluign_test_dartonlyplugin`
`Windows pluign_test_ plugin_ffi`
We have been spending effort to reduce the overall execution time of targets.

Reply:
Another look at the execution time trend: https://ci.chromium.org/p/flutter/builders/staging/Windows%20plugin_test. The total time is roughly around the 30 mins range.
Instead of three, maybe two is enough.

Reply:
fwiw in general i would be much more comfortable if none of our tests had special-cased timeouts. if a test is taking too long, we should either split it up into multiple tests or speed up the test.